meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "release": "1",
+    "epoch": "1",
+    "maintainer": "Serokell <hi@serokell.io>"
```

Comment:
Why did you change "Serokell https://serokell.io <hi@serokell.io>" to "Serokell <hi@serokell.io>"?

Reply:
`dh_make` doesn't like links and likes emails in this format:(